src/common.css

```diff
@@ -190,7 +194,8 @@ body > .select2-container.open {
 }
 
 /* See Scrollable Menu with Bootstrap 3 http://stackoverflow.com/questions/19227496 */
-.ui-select-bootstrap > .ui-select-choices ,.ui-select-bootstrap > .ui-select-no-choice {
+.ui-select-container > .ui-select-choices-bootstrap,
```

Comment:
this is going to break the non bootstrap usage of ui-select, right? so at this point we're admitting we're never going to try and get this stuff pushed upstream?